modules/TodoAppPage/components/DisplayTask/DisplayTask.tsx

```diff
@@ -100,11 +87,21 @@ function DisplayTask({
 
             <Text size="sm" c="dimmed" mb={4}>
               Priority:{" "}
-              {task.priority === EPriority.LOW
-                ? "Low"
-                : task.priority === EPriority.MEDIUM
-                ? "Medium"
-                : "High"}
+              <Badge
+                color={
+                  task.priority === EPriority.LOW
```

Comment:
Move to a function. Looks like you did similar things to multiple places. We can create a helper function and then use it everywhere required.